doc/DeveloperGuide.md

```diff
@@ -87,6 +87,42 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Sort persons
+
+**MSS**
+
+1. User requests to sort persons
+2. AddressBook shows a list of persons sorted in the alphabetical order of names.
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
```

Comment:
At least the system needs to inform the user that there are no people in the Addressbook before ending.